.github/workflows/ci-workflow.yml

```diff
@@ -44,6 +44,9 @@ jobs:
         - name: bitcoin
           repo: app-bitcoin-new
           branch: develop
+        - name: solana
+          repo: app-solana
+          branch: wip/fbeutin/swap_rebase_post_refacto
```

Comment:
I guess to be updated once Solana code is ready.

Reply:
Yep, I'll keep the comment open to keep this in mind